configs/face_2d_keypoint/topdown_regression/wflw/td-reg_res50_8x64e-210e_wflw-256x256.py

```diff
@@ -0,0 +1,121 @@
+_base_ = [
+    '../../../_base_/default_runtime.py', '../../../_base_/datasets/wflw.py'
```

Comment:
```suggestion
    '../../../_base_/default_runtime.py'
```

Reply:
> 

Yes, this is redundant. I removed it in the latest commit.